org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/DateTestCase.xtend

```diff
@@ -1,13 +1,31 @@
 package org.uqbar.project.wollok.tests.interpreter
 
-import org.uqbar.project.wollok.tests.interpreter.AbstractWollokInterpreterTestCase
+import java.time.LocalDate
 import org.junit.Test
 
 /**
  * 
  * @author dodain
  */
 class DateTestCase extends AbstractWollokInterpreterTestCase {
+
+	@Test
+	def void defaultDateCreationInitializesItsAttributesOk() {
+		'''
+		const now = new Date()
+		assert.equals(now.day(), «LocalDate.now().dayOfMonth»)
+		assert.notThat(now.day() === null)
+		'''.test
+	}
+	
+	@Test
+	def void initializedDateCreationInitializesItsAttributesOk() {
+		'''
+		const otroNow = new Date(day = 1, month = 2, year = 2019)
+		assert.notThat(otroNow.day() === null)
```

Comment:
Este `assert` no es redundante ? ya que abajo estamos testeando que lo que devuelve este metodo es igual a 1 ?
Lo mismo con el test de arriba

Reply:
Para reforzar el test, no estaria bueno chequear por el `month` y `year` que le pasamos por parametro ?
Aunque como nice to have estaria copado que esten en tests separados asi quedan mas legibles, tengo que ponerme a investigar sobre como meter `DisplayName`  aca :rocket:

Reply:
Sí, tenés razón que es redundante. Comencé testeando que no fuera `null` y después empecé a agregar más tests, en un refactor los separé pero no me avivé que no era neceesario. Ahora los mejoro, gracias @Guusy!

Y sería buenísimo ver cómo inyectarle una annotation DisplayName para de a poco llevarlo a un formato rspec...

Reply:
Buenisimo !

No conozco mucho el proyecto aun pero veo que usamos junit, si migramos a la version 5
podemos usar [display names](https://junit.org/junit5/docs/current/user-guide/#writing-tests-display-names) que tiene la annotation `@DisplayName("un nombre re copado")`

Reply:
+1 a subir la versión de JUnit a 5 (ya vi que se banca JDK 1.8) y usar Display Name, obvio que gradualmente (ir generando nuevos tests o migrar muuuuuy de a poco cuando tengas que tocar algo). Qué opinan @npasserini @PalumboN ?

Reply:
Sisis obvio, me parece que es lo mas sano ! 
Que buena noticia ! es tan grande el repo que no sabia donde estaba la version de junit jajaaj